src/cbc_sdk/audit_remediation/base.py

```diff
@@ -1413,7 +1413,9 @@ def async_export(self):
         """
         Create an asynchronous job that exports the results from the run.
 
-        This is recommended if you are expecting a very large result set.
+        This is recommended if you are expecting a very large result set.  Once the Job is created, wait for it to be
+        completed (using its await_completion() method or similar), then get the results from the Job using one of the
```

Comment:
What is `or similar`? I would expand this to indicate that you can write your own logic to wait for the state to be marked completed. Preferably adding a snippet of sample code

Reply:
I'll expand that to say `or some other method that waits for the status of the job to be completed`, but let's not clutter up this docstring by adding in sample code for something that doesn't even directly relate to `async_export()`.

Reply:
`some other method` - I'd ask what other functions can I use to wait? Really what you a recommending is that they write their own logic to wait for the job completion. It's valuable to reference the helper function but I would do just that as a recommendation. What about adding that after the sentence and embedding a link to that section in the text

```
        This is recommended if you are expecting a very large result set.  Once the Job is created, wait for it to be
        completed, then get the results from the Job using one of the get_output_ methods on the Job object. If need to 
        wait for the results asynchronously use the [await_completion()](link....) method in the Job object.
```

Reply:
OK, I rephrased it again.